src/main/com/example011/radek/v1/Mines.java

```diff
@@ -0,0 +1,138 @@
+package main.com.example011.radek.v1;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Scanner;
+
+
+public class Mines {
+
+    protected int playgroundSize;
+
+    protected int numberOfMines;
+
+    protected String[][] playground;
+
+    public Mines(int size, int mines) {
+
+        if (setPlaygroundSize(size)) {
+            setNumberOfMines(mines);
+        } else {
+            System.out.println("Bad size of playground, size must be bigger then 5!");
+            System.exit(0);
+        }
+    }
+
+    public void fillPlaygroundWithMines() throws NoSuchAlgorithmException {
+
+        playground = new String[playgroundSize][playgroundSize];
+        SecureRandom random = SecureRandom.getInstanceStrong();
+        int placedMines = 0;
+        while (placedMines != getNumberOfMines() || placedMines == 0) {
+            int generatedRowNumber = random.nextInt(playgroundSize);
+            int generatedColumnNumber = random.nextInt(playgroundSize);
+            if (playground[generatedRowNumber][generatedColumnNumber] == null) {
+                playground[generatedRowNumber][generatedColumnNumber] = "x";
+                placedMines = placedMines + 1;
+            }
+        }
+    }
+
+    public void printPlayground(boolean showMines) {
+
+        System.out.println("Generating playground...");
+        StringBuilder printMessageBuilder = new StringBuilder(playgroundSize * playgroundSize * 3);
+        for (int i = 0; i < playgroundSize; i++) {
```

Comment:
prvni for iterujes pomocí i, druhý pomocí l ... melo by to jit za sebou, pokud vnorujes fory do sebe ... takze i, j, k ....

Reply:
Jasně, budu dodržovat abecedu